system/sched_note/note_main.c

```diff
@@ -90,27 +90,35 @@ static FAR const char *g_statenames[] =
 static void trace_dump_unflatten(FAR void *dst,
                                  FAR uint8_t *src, size_t len)
 {
+#ifdef CONFIG_HAVE_LONG_LONG
+  uint64_t tmp = 0;
+#else
+  uint32_t tmp = 0;
+#endif
+
   switch (len)
     {
 #ifdef CONFIG_HAVE_LONG_LONG
       case 8:
-        *(uint64_t *)dst = ((uint64_t)src[7] << 56)
-                         + ((uint64_t)src[6] << 48)
-                         + ((uint64_t)src[5] << 40)
-                         + ((uint64_t)src[4] << 32);
+        tmp += ((uint64_t)src[7] << 56);
+        tmp += ((uint64_t)src[6] << 48);
+        tmp += ((uint64_t)src[5] << 40);
+        tmp += ((uint64_t)src[4] << 32);
```

Comment:
let's follow the same change:
https://github.com/apache/incubator-nuttx/pull/5992